### DISCUSSION TITLE
phone number support?

### DISCUSSION THREAD

**Discussion by nicolasburtey** [ory/kratos#2002] [Q&A] [answered]
I'm not sure understand if phone number are fully supported in Kratos. Some [issues are opened since 2019](https://github.com/ory/kratos/issues/137) related to it, but by looking at the [documentation](https://www.ory.sh/kratos/docs/concepts/credentials/username-email-password/#use-case-phone-number-and-password), phone is being mentioned so I'm assuming there is only partial support for phone?

what is compatible and not compatible with phones?

**Comment by aeneasr** [comment, marked as answer]
There's two PRs open right now, once they're merged it will be supported.

**Reply by vinckr** [reply]
For reference @nicolasburtey 
https://github.com/ory/kratos/pull/1941
https://github.com/ory/kratos/pull/1938

**Reply by nicolasburtey** [reply]
thanks I have seen them, thanks. hopefully those PRs are not stale

**Reply by lucky-lusa** [reply]
It seems that the PRs have now been merged. However, I still get errors ("received unexpected via type: phone"). Is the feature already in the 0.10.1 release? 

Is there a documentation somewhere? My concern is that users can register with username and optionally 1-n phone numbers or emails.
